pybind11_stubgen/__init__.py

```diff
@@ -470,20 +480,21 @@ def strip_current_module_name(obj, module_name):
             "class {class_name}({base_classes_list}):{doc_string}".format(
                 class_name=self.klass.__name__,
                 base_classes_list=", ".join(base_classes_list),
-                doc_string='\n' + self.INDENT + '"""{}"""'.format(self.doc_string) if self.doc_string else "",
+                doc_string='\n' + self.indent_lines('"""\n{}\n"""'.format(self.doc_string.strip("\n")))
+                            if self.doc_string else "",
             ),
         ]
         for f in self.methods:
             if f.name not in self.methods_blacklist:
-                result.extend(map(self.indent, f.to_lines()))
+                result.extend(map(self.indent_lines, f.to_lines()))
```

Comment:
`map` is redundant now

Reply:
`indent_lines` was wrong. So `self.indent` is back and `map` should stay.